docs/docusaurus/docs/core/run_validations/_choose_result_format/_SUMMARY.md

```diff
@@ -0,0 +1,9 @@
+When the `result_format` key is set to `"SUMMARY"` the Validation Results of each Expectation includes a `result` dictionary with information that summarizes values to justify why it failed or succeeded.  This format is intended for more detailed exploratory work and includes additional information beyond what is included by `BASIC`.
```

Comment:
nit: Is "justify" the right word? Maybe "explain"?

Reply:
"Justify" is a hold over from the original language in the Result Format docs.  I think I also prefer "explain," and will update accordingly.